packages/keyring/src/pair/index.spec.ts

```diff
@@ -171,20 +154,36 @@ describe('pair', (): void => {
 
     // decrypt: Bob-ed25519 - use alice-ed25519 pubkey
     expect(
-      keyring.bob.decryptMessage(encrypted3, keyring.alice.publicKey)
+      keyring.bob.decryptMessage(encrypted3)
     ).toEqual(message);
 
     // decrypt: Bob-ed25519 - use alice-sr25519 pubkey
     expect(
-      keyring.bob.decryptMessage(encrypted1, pairA.publicKey)
+      keyring.bob.decryptMessage(encrypted1)
     ).toEqual(message);
 
     // decrypt: Bob-sr25519 - use alice-sr25519 pubkey
     expect(
-      pairB.decryptMessage(encrypted2, pairA.publicKey)
+      pairB.decryptMessage(encrypted2)
     ).toEqual(message);
   });
 
+  it('passes the test on issue#1314', (): void => {
+    // Ref: https://github.com/polkadot-js/common/issues/1314
+
+    const keyring = new Keyring();
+
+    const senderPair = keyring.createFromUri(mnemonicGenerate(), { name: 'first pair' }, 'ed25519');
+    const receiverPair = keyring.createFromUri(mnemonicGenerate(), { name: 'second pair' }, 'ed25519');
```

Comment:
Unfortunately, the issue #1314 talks about `sr25519`, the previous code was ok with `ed25519`.

Reply:
Uh. Sorry! Forget to change this in testing. Updating soon.